V2.23.0: Potentially broken release?

### How do you use Sentry?

Self-hosted/on-premise

### Version

2.23

### Steps to Reproduce

1. Upgrade sentry to 2.23
2. use `from sentry_sdk import *`

### Expected Result

The import should just work.

### Actual Result

```
E   AttributeError: module 'sentry_sdk' has no attribute '_experimental_logger.py'. Did you mean: '_experimental_logger'?
```

It worked fine with 2.22.

--- a/sentry_sdk/__init__.py
+++ b/sentry_sdk/__init__.py
@@ -45,7 +45,7 @@ __all__ = [  # noqa
     "start_transaction",
     "trace",
     "monitor",
-    "_experimental_logger.py",
+    "_experimental_logger",
 ]
 
 # Initialize the debug support after everything is loaded

--- a/tests/test_import.py
+++ b/tests/test_import.py
@@ -1,9 +1,7 @@
-
 # As long as this file can be imported, we are good.
-from sentry_sdk import *
+from sentry_sdk import *  # noqa: F403, F401
 
 
 def test_import():
     # As long as this file can be imported, we are good.
     assert True
-    

--- a/tests/test_import.py
+++ b/tests/test_import.py
@@ -1,0 +1,9 @@
+
+# As long as this file can be imported, we are good.
+from sentry_sdk import *
+
+
+def test_import():
+    # As long as this file can be imported, we are good.
+    assert True
+    

